Phase cam6110 new version

ChiaraSelmi opened the pull request:
Modification of the current code to have the client-server structure also for the interferometers that use WCF interface for 4Sight software in order to add the possibility to acquire burst frame using the burst acquisition function provide by 4D. This function has been implemented in the abstract interferometers and also added in all actual devices.

Concept: local acquisition (pc interferometer) of raw files and local conversion into .4D files. Files re-reading and averaging. Output: mean_image
Tested using the Padova interferometer.

lbusoni commented:
@ChiaraSelmi I provided a framework for testing the WCFInterfacer, by overriding the _readJsonData method. You can extend the tests, now it is testing only the method wavefront() with how_many=1.
When you are happy with testing can we merge and close this branch? And the similar one in plico_interferometer?

ChiaraSelmi commented:
Code tested with the AccuFitz.
Implemented commmand are:
- wavefront: if a single measurement is required the function uses the standard command for acquisition, in the other cases it uses the burst functionality. A temporary folder is created and deleted after it's use.
- acquire burst: a tracking number with the raw files is generated on the 4d pc (file path specified by the user in the configuration file)
- load burst: create the data folder with the .4D files anche returns the cube
- delete burst: delete the tracking number desired
- list available burst: returns the list of burst folder available.
